### ISSUE TITLE
TS discontinuity errors

### ISSUE THREAD

**Issue by macstr1k3r** [sergey-dryabzhinsky/nginx-rtmp-module#64]
When re-streaming hls streams generated by nginx-rtmp-module with vlc the vlc log is filled with messages like the ones bellow

ts error: libdvbpsi (PSI decoder): TS duplicate (received 0, expected 1) for PID 0
ts error: libdvbpsi (PSI decoder): TS duplicate (received 0, expected 1) for PID 4095
ts error: libdvbpsi (PSI decoder): TS duplicate (received 0, expected 1) for PID 0
ts error: libdvbpsi (PSI decoder): TS duplicate (received 0, expected 1) for PID 4095

Basically this should be happening when the continuity counter wraps.

**Comment by premultiply** [comment]
For me it looks like ngx_rtmp_mpegts.c does not update the continuity_counter for packets pat (pid 0) and pmt packets (pid 4095). They are currently just static zero but need to be updated on every packet.

**Comment by sergey-dryabzhinsky** [comment]
Can you please try code from ```64i-mpegts-continuity-counter-fix``` branch?

**Comment by sergey-dryabzhinsky** [comment]
It's merged to master branch.

**Comment by BradDodi** [comment]
I'm also experiencing this issue with the latest master branch. The message log is filled with errors such as those in the original post.
